config.py

```diff
@@ -16,6 +31,11 @@ def getenv(env_var, default=NOT_PROVIDED):
         raise Exception(f"Environment Variable '{env_var}' not set and no default provided")
     return value
 
+def getenv_unique_list(var_name) -> set[str]:
```

Comment:
(minor) it returns a set, so I would skip "list" from the name perhaps? => `getenv_unique`

Reply:
Renamed to `getenv_unique_set` since returning a non-string from an environment variable might surprise people